### PR TITLE
feat: show email list subscription

### PR DESCRIPTION
**Pull request by petrjasek**
on first login.

SDESK-521

requires https://github.com/superdesk/superdesk-core/pull/753

### CODE REVIEW

**Comment by naspeh** [comment]
@petrjasek probably we need this fix for `"1.5.9"`  in separate commit to cherry-pick it to other branches (like 1.5)

**Comment by naspeh** [comment]
It just needs `angular-mocks@1.6.1` to fix `npm test`, #1322 is green already!

**Comment by hlmnrmr** [comment]
@naspeh This is a new feature, it should not be backported

**Comment by naspeh** [comment]
@hlmnrmr I told about fix for `npm test`, @petrjasek fixed them in this PR, so I suggested to create separate commit for this, but then I've fixed `npm test` in #1322.

File: package.json [hunk 1]
```diff
@@ -29,19 +29,19 @@
   "main": "scripts/index.js",
   "dependencies": {
     "angular": "1.5.9",
-    "angular-animate": "^1.5.5",
+    "angular-animate": "1.5.9",
```
Comment (naspeh) [inline package.json]: Please revert these versions, `npm test` is working now on master

**Pull request merged**